### DISCUSSION TITLE
Add DLLs for rulesets in "Import Files" filetypes

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#20473] [Ideas] [answered]
In some platforms, like mobile, it is extremely annoying to have to search manually the folder for osu in your device, and then try to move it, and not be able to. And end up having to use your computer, or in iOS devices, near impossible to add the DLLs

**Comment by nagi-desuuu** [comment, marked as answer]
It would be impossible to load rulesets into mobile platforms anyway because they don't support dynamic code loading, unlike Windows.

**Reply by peppy** [reply]
As mentioned, we can probably add support for loading rulesets via drag/import, but it isn't going to help you on mobile.
